app/filters/MicroserviceAuthFilter.scala

```diff
@@ -25,15 +25,15 @@ import play.api.routing.Router
 import uk.gov.hmrc.auth.core.{AuthorisationException, AuthorisedFunctions}
 import connectors.AuthConnector
 import controllers.ErrorUnauthorized
-import uk.gov.hmrc.play.HeaderCarrierConverter
+import uk.gov.hmrc.play.http.HeaderCarrierConverter
 
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class MicroserviceAuthFilter @Inject() (configuration: Configuration, val authConnector: AuthConnector)(implicit val mat: Materializer, ec: ExecutionContext) extends Filter with AuthorisedFunctions with Results {
 
   def apply(next: (RequestHeader) => Future[Result])(rh: RequestHeader): Future[Result] = {
-    implicit val hc = HeaderCarrierConverter.fromHeadersAndSession(rh.headers)
+    implicit val hc = HeaderCarrierConverter.fromRequestAndSession(rh, rh.session)
```

Comment:
this might need to be `fromRequest` instead

Reply:
Done 👍